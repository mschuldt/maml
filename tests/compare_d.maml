arduino = Arduino(desktop=True)

@block(once)
def test():
    x = 0
    print_i(x < 3)
    print_i(2 < x)
    print_i(1 < 2)
    print_i(2 < 1)
    die()

arduino.send(test)
