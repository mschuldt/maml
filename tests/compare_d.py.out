Initializing avm...
Ready.

1
0
1
0
bye.
